builtins/methods/pyint/pyint.go

```diff
@@ -0,0 +1,44 @@
+package pyint
+
+import (
+	"math/bits"
+	"strconv"
+)
+
+type PyInt int
+
+func New(i int) PyInt {
+	return PyInt(i)
+}
+
+func (pi PyInt) Int() int {
+	return int(pi)
+}
+
+func (pi PyInt) String() string {
+	return strconv.Itoa(int(pi))
+}
+
+func (pi PyInt) BitLength() int {
+	return bits.Len(uint(pi))
+}
+
+func (pi PyInt) BitCount() int {
+	return bits.OnesCount(uint(pi))
+}
+
+func (pi PyInt) AsIntegerRatio() (int, int) {
+	return int(pi), 1
+}
+
+func (pi PyInt) IsInteger() bool {
+	return true
+}
+
+//func (pi PyInt) ToBytes(length=1, byteorder='big', *, signed=False) {
+//
+//}
+
+//func (pi PyInt) FromBytes(bytes, byteorder='big', *, signed=False) {
+//
+//}
```

Comment:
Why is this commented out ?

Reply:
I semi-automatically generated the function heads from python docs; Didn't found time to implement them yet

Reply:
does gonja have a native type for bytes? I can't seem to find such support meaning that conversions to / from bytes can't be exposed as a method. 

The conversion logic is in place but won't be able to add them as pyint methods.

Reply:
> does gonja have a native type for bytes? 
@Zatte no it does not and it would require some work to add it in. Maybe for later, but this PR is already quite huge so let's proceed with baby steps